Emerald Insight.js

```diff
@@ -0,0 +1,240 @@
+				});
+			}
+			item.complete();
+		});
+		translator.translate();
+	});
+}
+
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "https://www.emerald.com/insight/content/doi/10.1108/IJPH-07-2016-0028/full/html",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Universal opt-out screening for hepatitis C virus (HCV) within correctional facilities is an effective intervention to improve public health",
+				"creators": [
+					{
+						"lastName": "Morris",
+						"firstName": "Meghan D.",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Brown",
+						"firstName": "Brandon",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Allen",
+						"firstName": "Scott A.",
+						"creatorType": "author"
+					}
+				],
+				"date": "January 1, 2017",
+				"DOI": "10.1108/IJPH-07-2016-0028",
+				"ISSN": "1744-9200",
+				"abstractNote": "Purpose Worldwide efforts to identify individuals infected with the hepatitis C virus (HCV) focus almost exclusively on community healthcare systems, thereby failing to reach high-risk populations and those with poor access to primary care. In the USA, community-based HCV testing policies and guidelines overlook correctional facilities, where HCV rates are believed to be as high as 40 percent. This is a missed opportunity: more than ten million Americans move through correctional facilities each year. Herein, the purpose of this paper is to examine HCV testing practices in the US correctional system, California and describe how universal opt-out HCV testing could expand early HCV detection, improve public health in correctional facilities and communities, and prove cost-effective over time.Design/methodology/approach A commentary on the value of standardizing screening programs across facilities by mandating all facilities (universal) to implement opt-out testing policies for all prisoners upon entry to the correctional facilities.Findings Current variability in facility-level testing programs results in inconsistent testing levels across correctional facilities, and therefore makes estimating the actual number of HCV-infected adults in the USA difficult. The authors argue that universal opt-out testing policies ensure earlier diagnosis of HCV among a population most affected by the disease and is more cost-effective than selective testing policies.Originality/value The commentary explores the current limitations of selective testing policies in correctional systems and provides recommendations and implications for public health and correctional organizations.",
+				"issue": "3/4",
+				"libraryCatalog": "Emerald Insight",
+				"pages": "192-199",
+				"publicationTitle": "International Journal of Prisoner Health",
+				"url": "https://doi.org/10.1108/IJPH-07-2016-0028",
+				"volume": "13",
+				"attachments": [
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [],
```

Comment:
There are keywords which could be saved as tags, i.e. look for `doc.querySelectorAll('li .intent_text');`.

Reply:
Done. Won't use when scraping from PDF view which I think is acceptable.